### ISSUE TITLE
Can't exclude issues without labels

### ISSUE THREAD

**Issue by spacecowboy** [github-changelog-generator/github-changelog-generator#302]
Running with `--no-issues-wo-labels` seems to make no difference. There is still a `Closed issues` section generated with issues that have no tags.

**Comment by chros73** [comment]
I can confirm this issue with `--no-issues-wo-labels` and the same applies to `--no-pr-wo-labels` as well (no matter whether `--no-filter-by-milestone` is set or not).

Sample config:
```
compare-link=false
issues-wo-labels=false
pr-wo-labels=false
unreleased=true
future-release=1.4-0.9.6
exclude-tags=final-0.8.x-support
since-tag=0.9.4-20150310-46ed5cb
```

URL to issues: https://github.com/chros73/rtorrent-ps/issues?q=is%3Aissue+is%3Aclosed
Note: There are 3 issues without label and all of them are included (every issue has been assigned to a milestone (there's no issue with multiple label)). Also it doesn't matter whether an issue that hasn't got label is assigned to a milestone or not.

URL to pull requests: https://github.com/chros73/rtorrent-ps/pulls?q=is%3Apr+is%3Aclosed
Note: 2 of them out of 3 is included:

```
- Update to current head fork [\#7](https://github.com/chros73/rtorrent-ps/pull/7) ([chros73](https://github.com/chros73))
- Merging changes from original master branch [\#6](https://github.com/chros73/rtorrent-ps/pull/6) ([chros73](https://github.com/chros73))
```

What can be the quick workaround for excluding simple closed issues?

Thanks

**Comment by chros73** [comment]
So, the workaround for these bugs for now can be that we assign a dummy label (e.g. `no-changelog`) to those issues/pull-requests and use `exclude-labels=no-changelog` in config.
The working config (with couple of hacks :) ) is:
```
compare-link=false
#issues-wo-labels=false
#pr-wo-labels=false
exclude-labels=no-changelog
unreleased=true
future-release=1.4-0.9.6
exclude-tags=final-0.8.x-support
since-tag=0.9.4-20150310-46ed5cb
```

**Comment by mvz** [comment]
Like #292, this was probably fixed in #618.
